CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/player/expand/ExpandPlayer.java

```diff
@@ -36,6 +37,11 @@ public static IContainer getInventoryContainer(IPlayer player) {
         return CraftTweakerMC.getIContainer(getInternal(player).inventoryContainer);
     }
 
+    @ZenMethod
+    public static void sendRichTextStatusMessage(IPlayer player, ITextComponent textComponent, boolean hotbar) {
```

Comment:
Better make the hotbar argument optional, so that the player can leave it out (which is the case if they wanted to use the method from ExpandCommandSender).